index/scorch/snapshot_index_tfr.go

```diff
@@ -146,6 +162,27 @@ func (i *IndexSnapshotTermFieldReader) Advance(ID index.IndexInternalID, preAllo
 	return next, nil
 }
 
+// glean expects preAlloced TermFieldDoc
+func (i *IndexSnapshotTermFieldReader) glean(ID index.IndexInternalID,
```

Comment:
In the category of "naming is difficult", I think the method name "glean" probably needs a better name :-)